### PR TITLE
Added osbs profile with and checks for required labels and description

### PR DESCRIPTION
**Pull request by dav1x**

### CODE REVIEW

File: sample_rules/osbs.yaml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+             valueRegex: /([\w]+.*)./
+             message: "Label 'Release' is missing or has invalid format"
+             level: "error"
+             required: true
+             reference_url:
+               - "http://docs.projectatomic.io/container-best-practices/#"
+               - "_recommended_labels_for_your_project"
+           Architecture:
+             valueRegex: /[\w]*[6,8][4,6]|[.]*86[.]*64/
+             message: "Label 'Architure' is missing or has invalid format: x86, i386, x86_64"
```
Comment (twaugh) [inline sample_rules/osbs.yaml]: Typo

File: sample_rules/osbs.yaml [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            - "https://docs.docker.com/reference/builder/"
+            - "#from"
+          label: "specified_registry"
+          regex: "/[/]/"
+          level: "warn"
+          message: "using a specified registry in the FROM line"
+          description: "using a specified registry may supply invalid or unexpected base images"
+          reference_url:
+            - "https://docs.docker.com/reference/builder/"
+            - "#entrypoint"
```
Comment (twaugh) [inline sample_rules/osbs.yaml]: The special FROM value koji/image-build (with or without a tag suffix) should be allowed as that is how base images can be built.

@lcarva - do we have documentation for that yet?

Comment (twaugh) [inline sample_rules/osbs.yaml]: Seems not. https://github.com/projectatomic/atomic-reactor/issues/510

Comment (dav1x) [inline sample_rules/osbs.yaml]: Changed the tag search to a warning vs. error.

**Pull request merged**